spec/helpers/nav_helper_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe NavHelper, type: :helper do
```

Comment:
You're right. The original point of this was to test whether the inner block was called (which is equivalent to check for the existance of a link inside a list-item). I can just look for `li > a`